CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+cmake_minimum_required(VERSION 3.1)
+project(rhash)
+
+file(READ "${CMAKE_CURRENT_SOURCE_DIR}/version.h" versionfile)
+string(REGEX MATCH "#define VERSION \"([0-9]*)\.([0-9]*)\.([0-9]*)\"" _ ${versionfile})
+set(RHASH_VERSION_MAJOR ${CMAKE_MATCH_1})
+set(RHASH_VERSION_MINOR ${CMAKE_MATCH_2})
+set(RHASH_VERSION_PATCH ${CMAKE_MATCH_3})
+set(RHASH_VERSION "${RHASH_VERSION_MAJOR}.${RHASH_VERSION_MINOR}.${RHASH_VERSION_PATCH}")
+
+option(USE_GETTEXT "Enable gettext (localization) support")
+
+set(SOURCE_FILES    "${CMAKE_CURRENT_SOURCE_DIR}/calc_sums.c"
```

Comment:
no need for CMAKE_CURRENT_SOURCE_DIR, files are always relative to the source dir